.circleci/config.yml

```diff
@@ -39,8 +39,18 @@ jobs:
             rustup component add rustfmt
             cargo fmt --all -- --check
 
+  coverage:
+    machine: true
+    steps:
+        - checkout
+        - run:
+            name: Coverage with docker
+            command: docker run --security-opt seccomp=unconfined -v "${PWD}:/volume" xd009642/tarpaulin sh -c "cargo tarpaulin --ciserver circle-ci --coveralls $COVERALLS_REPO_TOKEN"
```

Comment:
Just a shame that we have to do it this "ugly" way, but I see on the repo it is the suggested way, so who am I to complain. :man_shrugging: 
Thanks for adding the code coverage :+1:

Reply:
Code coverage needs special access to augment the tests for reporting.